app/javascript/locales/es.js

```diff
@@ -33,7 +33,8 @@ export default {
       registerTitle: 'Crear cuenta',
       loginTitle: 'Iniciar sesión',
       labelEmail: 'Correo',
-      labelPassword: 'Nueva contraseña',
+      labelPassword: 'Contraseña',
+      labelNewPassword: 'Nueva contraeña',
```

Comment:
```suggestion
      labelNewPassword: 'Nueva contraseña',
```
Faltaba una _s_ jeje